scripts/monorepo/bump-all-updated-packages/bump-package-version.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+const {writeFileSync, readFileSync} = require('fs');
+const path = require('path');
+
+const forEachPackage = require('../for-each-package');
+
+const ROOT_LOCATION = path.join(__dirname, '..', '..', '..');
+const TEMPLATE_LOCATION = path.join(ROOT_LOCATION, 'template');
+const REPO_CONFIG_LOCATION = path.join(ROOT_LOCATION, 'repo-config');
+
+const readJSONFile = packageAbsolutePath =>
+  JSON.parse(readFileSync(packageAbsolutePath));
+
+const getIncrementedVersion = (version, increment) =>
```

Comment:
Is there a reason for why we can't use [Changesets](https://github.com/changesets/changesets) (or even [Beachball](https://microsoft.github.io/beachball/)) instead of writing our own version bumping script? There are existing tools that are specialized for this purpose.

Reply:
talked with Tommy separately and I've explained to him why we need custom scripts in this phase